nixos/modules/services/web-apps/healthchecks.nix

```diff
@@ -191,8 +208,34 @@ in
   };
 
   config = mkIf cfg.enable {
+    assertions = [
+      {
+        assertion = cfg.configurePostgres -> cfg.user == defaultUser;
```

Comment:
Why do you restrict it to the default user "healtchecks"?

I think if we are adding a layer like, mostly to ensure the existence of the DB and user, we could leave the user customizable with no cost. There's already an option here, using that might result in even code than adding an extra assertion for that case.

Reply:
That's mostly because the database name and user must match (for permissions and support for differences got removed from postgres) and in many modules that it js just hardcoded to make things simpler.

I can change the assertion so that user and DB must match. That would allow a bit more flexibility.

Reply:
I understand that they must match, but we do control both here, so I don't see the problem?